.travis.yml

```diff
@@ -0,0 +1,40 @@
+language: android
+
+jdk:
+ - oraclejdk7
+ - oraclejdk8
+
+matrix:
+  allow_failures:
+    - jdk: oraclejdk7
```

Comment:
Depending on your local system and memory, it may not. I just added that to show that it "can" compile with jdk7. We can probably remove it if you do not want it.